src/nft/components/bag/profile/ListingButton.tsx

```diff
@@ -158,94 +181,108 @@ export const ListingButton = ({ onClick, buttonText, showWarningOverride = false
-          flexWrap={!disableListButton ? 'wrap' : 'nowrap'}
-          style={{ maxWidth: !disableListButton ? '225px' : '' }}
-          ref={warningRef}
-        >
-          <HazardIcon />
-          <Box marginLeft="4" marginRight="8">
-            {warningMessage}
-          </Box>
-          {disableListButton ? (
-            <Box paddingTop="6">
-              <XMarkIcon fill={themeVars.colors.textSecondary} height="20" width="20" />
+    <>
+      <Box position="relative" width="full">
+        {!showWarningOverride && showWarning && warningMessage.length > 0 && (
+          <Row
+            className={`${bodySmall} ${styles.warningTooltip}`}
+            transition="250"
+            onClick={() => setShowWarning(false)}
+            color="textSecondary"
+            zIndex="3"
+            borderRadius="4"
+            backgroundColor="backgroundSurface"
+            height={!disableListButton ? '64' : '36'}
+            maxWidth="276"
+            position="absolute"
+            left="24"
+            bottom="52"
+            flexWrap={!disableListButton ? 'wrap' : 'nowrap'}
+            style={{ maxWidth: !disableListButton ? '225px' : '' }}
+            ref={warningRef}
+          >
+            <HazardIcon />
+            <Box marginLeft="4" marginRight="8">
+              {warningMessage}
             </Box>
-          ) : (
-            <Row
-              marginLeft="72"
-              cursor="pointer"
-              color="accentAction"
-              onClick={() => {
-                setShowWarning(false)
-                setCanContinue(true)
-                onClick()
-              }}
-            >
-              Continue
-              <ArrowRightIcon height="20" width="20" />
-            </Row>
-          )}
-        </Row>
-      )}
-      <Box
-        as="button"
-        border="none"
-        backgroundColor="accentAction"
-        cursor={
-          [ListingStatus.APPROVED, ListingStatus.PENDING, ListingStatus.SIGNING].includes(listingStatus) ||
-          disableListButton
-            ? 'default'
-            : 'pointer'
-        }
-        color="explicitWhite"
-        className={styles.button}
-        onClick={() => listingStatus !== ListingStatus.APPROVED && warningWrappedClick()}
-        type="button"
-        style={{
-          opacity:
-            ![ListingStatus.DEFINED, ListingStatus.FAILED, ListingStatus.CONTINUE].includes(listingStatus) ||
-            disableListButton
-              ? 0.3
-              : 1,
-        }}
-      >
-        {listingStatus === ListingStatus.SIGNING || listingStatus === ListingStatus.PENDING ? (
-          <Row gap="8">
-            <LoadingIcon stroke="backgroundSurface" height="20" width="20" />
-            {listingStatus === ListingStatus.PENDING ? 'Pending' : 'Proceed in wallet'}
+            {disableListButton ? (
+              <Box paddingTop="6">
+                <XMarkIcon fill={themeVars.colors.textSecondary} height="20" width="20" />
```

Comment:
tech debt: a few of these could be replaced with react feather - which would be great to clean up